### PR TITLE
ACM-2360: Add more steps to host inventory quick start

### PR DESCRIPTION
**Pull request by chenz4027**
Signed-off-by: Maggie Chen <magchen@redhat.com>
https://issues.redhat.com/browse/ACM-2360

### CODE REVIEW

**Review by matthewcarleton — APPROVED** [review]
LGTM

**Review by jhadvig — APPROVED** [review]
/approve
/lgtm

**Comment by chenz4027** [comment]
Needs more labels 

QE Approver:
 /assign @yanpzhan
Docs Approver:
 /assign @opayne1
PX Approver:
/assign @RickJWagner

File: quickstarts/host-inventory-quickstart.yaml [hunk 1]
```diff
@@ -21,9 +21,29 @@ spec:
 
 
   tasks:
-    - description: >-
+    - description: |-
 
-        1.  Configure your host inventory settings. View [documentation](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html-single/clusters/index#enable-cim) to follow the steps to configure your host inventory settings.
+        1.  Configure your host inventory settings.
+
+          a. Go to **Host inventory**.
```
Comment (opayne1) [inline quickstarts/host-inventory-quickstart.yaml:28]: Should we put where to go to get to host inventory?

File: quickstarts/host-inventory-quickstart.yaml [hunk 2]
```diff
@@ -21,9 +21,29 @@ spec:
...
+
+          b. Select **Configure host inventory settings**.
+
+        1. In the 'Configure host inventory settings' modal that opens,
```
Comment (opayne1) [inline quickstarts/host-inventory-quickstart.yaml:32]: I think you could combine step a with this task so it is formatted more like a task:
```suggestion
        1. Enter the required values for Database storage, System Storage, and Image Storage in the modal that opens. 
```

Comment (opayne1) [reply quickstarts/host-inventory-quickstart.yaml:32]: Also Database storage, System Storage, and Image Storage should be bolded based on https://docs.openshift.com/container-platform/4.11/web_console/creating-quick-start-tutorials.html#quick-start-content-guidelines-formatting-UI-elements_creating-quick-start-tutorials

File: quickstarts/host-inventory-quickstart.yaml [hunk 3]
```diff
@@ -21,9 +21,29 @@ spec:
...
+
+          a. Enter the required values for Database storage, System Storage, and Image Storage.
+
+          **Note**: The modal is for connected environments only. For disconnected environments, follow the steps in the [configuring your host inventory settings documentation](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html-single/clusters/index#enable-cim).
+
+          b. If your cluster is running on AWS and you have not configured your load balancer, select the 'Configure load balancer on Amazon Web Services for me' checkbox.
+
+        1. Then click configure.
+        
+          **Note**: Storage sizes cannot be changed after they are configured. Also, configuring the load balancer on AWS cannot be reverted.
```
Comment (opayne1) [inline quickstarts/host-inventory-quickstart.yaml:42]: Should this be two separate notes? If not I would remove the "Also"
```suggestion
          **Note**: Storage sizes cannot be changed after they are configured. 
           **Note**: Configuring the load balancer on AWS cannot be reverted.
```

File: quickstarts/host-inventory-quickstart.yaml [hunk 4]
```diff
@@ -21,9 +21,29 @@ spec:
...
+        
+        1. The configuration will take time. You'll be notified when the configuration is successfully complete.
```
Comment (opayne1) [inline quickstarts/host-inventory-quickstart.yaml:44]:
```suggestion
        1. The configuration will take time. You will be notified when the configuration is successfully complete.
```

File: quickstarts/host-inventory-quickstart.yaml [hunk 5]
```diff
@@ -21,9 +21,29 @@ spec:
...
+
+        1. Now your host inventory settings are successfully configured and you can create your first infrastructure environment.
```
Comment (opayne1) [inline quickstarts/host-inventory-quickstart.yaml:46]: I wonder if this task and the one above should be combined as notes since they aren't really tasks.

**Review by opayne1 — COMMENTED** [review]
Just a few comments. Mostly suggestions, but I think they would help with clarity. Let me know if you have questions! 😃

File: quickstarts/host-inventory-quickstart.yaml [hunk 6]
```diff
@@ -21,13 +21,31 @@ spec:
...
-        1.  Configure your host inventory settings. View [documentation](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html-single/clusters/index#enable-cim) to follow the steps to configure your host inventory settings.
+        1.  Configure your host inventory settings.
+
+          a. From the **All Clusters** view of the OpenShift Container Platform web console, navigate to the host inventory by clicking infrastructure > **Host inventory**
+
+          b. Select **Configure host inventory settings**.
+
+        1. Enter the required values for **Database storage**, **System Storage**, and **Image Storage** in the modal that opens. 
+
+          a. Enter the required values for Database storage, System Storage, and Image Storage.
```
Comment (opayne1) [inline quickstarts/host-inventory-quickstart.yaml:34]: Woops! looks like this step is still here after making it a task.

Comment (opayne1) [inline quickstarts/host-inventory-quickstart.yaml:34]: Woops! looks like this step is still here after making it a task.

File: quickstarts/host-inventory-quickstart.yaml [hunk 7]
```diff
@@ -21,13 +21,31 @@ spec:
...
+        1. Then click configure.
+
+          **Note**: Storage sizes cannot be changed after they are configured. Configuring the load balancer on AWS cannot be reverted.
+        
+        1. The configuration will take time. You will be notified when the configuration is successfully complete. Now your host inventory settings are successfully configured and you can create your first infrastructure environment.
 
         1. Create an infrastructure environment for your host inventory.
 
-          a. Go to **Host inventory**.
+          a. From the **All Clusters** view of the OpenShift Container Platform web console, navigate to the host inventory by clicking infrastructure > **Host inventory**
```
Comment (opayne1) [inline quickstarts/host-inventory-quickstart.yaml:48]: Should it be **Infrastructure**?
```suggestion
          a. From the **All Clusters** view of the OpenShift Container Platform web console, navigate to the host inventory by clicking **Infrastructure** > **Host inventory**
```

**Review by opayne1 — COMMENTED** [review]
Thanks for the update! Just noticed 2 more small things.

File: quickstarts/host-inventory-quickstart.yaml (hunk 6 above)
Comment (chenz4027) [reply quickstarts/host-inventory-quickstart.yaml]: Thanks @opayne1 , just changed it :)

**Review by opayne1 — COMMENTED** [review]
Thank you for making all the changes!
/label docs-approved

**Review by TheRealJon — APPROVED** [review]
/lgtm

**Comment by jhadvig** [comment]
label tide/merge-method-squash

**Comment by chenz4027** [comment]
Yep @jhadvig we would have to merge the cherry pick ones for the previous commits first

**Pull request merged**